regexes/client/mobile_apps.yml

```diff
@@ -272,19 +272,68 @@
   name: 'Procast'
   version: '$1'
 
+# DeviantArt (https://apps.apple.com/us/app/deviantart/id925219396)
+- regex: 'DeviantArt/([\d\.]+)'
+  name: 'DeviantArt'
+  version: ''
+
+# Google (https://apps.apple.com/us/app/google/id284815942)
+- regex: 'com.google.GoogleMobile/([\d\.]+)'
+  name: 'Google'
```

Comment:
I installed the application by the link, the useragent there is like this
'Mozilla/5.0 (iPhone; CPU iPhone OS 12_4_8 as Mac OS X) AppleWebKit/605.1.15 (KHTML as Gecko) GSA/137.2.345735309 Mobile/16G201 Safari/604.1`

set name `Google Search App`